src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/ClimbingPick.java

```diff
@@ -0,0 +1,147 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.tools;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import org.bukkit.Bukkit;
+import org.bukkit.GameMode;
+import org.bukkit.Material;
+import org.bukkit.Sound;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockFace;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.entity.Player;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.Vector;
+
+import io.github.thebusybiscuit.cscorelib2.materials.MaterialCollections;
+import io.github.thebusybiscuit.slimefun4.api.events.ClimbingPickLaunchEvent;
+import io.github.thebusybiscuit.slimefun4.api.items.ItemSetting;
+import io.github.thebusybiscuit.slimefun4.api.MinecraftVersion;
+import io.github.thebusybiscuit.slimefun4.core.attributes.DamageableItem;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Objects.handlers.BlockBreakHandler;
+import me.mrCookieSlime.Slimefun.Objects.handlers.ItemUseHandler;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+
+/**
+ * The {@link ClimbingPick} launches you 1 block upwards when you right click
+ * on a ice {@link Block}.
+ * Every level of efficiency {@link Enchantment} increases the launch by 0.2 blocks.
+ *
+ * @author Linox
+ *
+ */
+public class ClimbingPick extends SimpleSlimefunItem<ItemUseHandler> implements DamageableItem {
+
+    private final Map<String, Double> materialSpeedsDef = new HashMap<>();
+    private final ItemSetting<Map<String, Double>> materialSpeeds = new ItemSetting<>("launch-amounts", materialSpeedsDef);
```

Comment:
Do not use ItemSetting for complex data types, such as a Map...

Reply:
How can I fix this?

Reply:
By removing this ItemSetting.